Updated the links to open in new tab

HarvestingClouds opened the pull request:
Changed the links for "free account" and Azure portal to open in the new page. Markdown does not have this feature yet, so changed to normal html. The links opening in a new tab let's the reader be on the documentation page and the reader does not need to go back to the documentation if they need to follow the subsequent sections.

azurecla commented:
Hi __@YanZhangADS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (zhangya). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

HarvestingClouds committed:
Updated the links to open in new tab

Changed the links for "free account" and Azure portal to open in the new page. Markdown does not have this feature yet, so changed to normal html. The links opening in a new tab let's the reader be on the documentation page and the reader does not need to go back to the documentation if they need to follow the subsequent sections.
